app/models/queries/members/filters/entity_type_filter.rb

```diff
@@ -32,7 +32,10 @@ def type
   end
 
   def allowed_values
-    [[WorkPackage.name, WorkPackage.name]]
+    [
+      [WorkPackage.name, WorkPackage.name],
+      [ProjectQuery.name, ProjectQuery.name]
```

Comment:
Looks like double vision, I suggest either generating repetition or trying without repetition, as usages seem to call either first or last:
```suggestion
      [WorkPackage.name],
      [ProjectQuery.name]
```

Reply:
Orienting on other places where we use this as well